src/view/channel-detail.js

```diff
@@ -42,6 +42,11 @@ const ChannelDetailView = ({ store, nav }) => (
             {store.selectedChannel.closingTxId}
           </DetailField>
         ) : null}
+        {store.selectedChannel.blocksTilMaturity ? (
```

Comment:
This will always be falsy ... change to `store.selectedChannel.timeTilAvailable`

Reply:
fixed